.github/workflows/tacos_conflict.yml

```diff
@@ -0,0 +1,87 @@
+name: TACOS Conflict
+
+# downstream projects should use events:
+###pull_request_target:
+###  types:
+###    - closed
+###    - converted_to_draft
+###    - labeled
+###    - opened
+###    - synchronize
+###    - reopened
+###    - ready_for_review
+
+on:
+  workflow_call:
+    inputs:
+      tacos_gha_repo:
+        type: string
+        default: getsentry/tacos-gha
+      tacos_gha_ref:
+        type: string
+        default: refs/heads/stable
+
+defaults:
+  run:
+    shell: env ./tacos-gha/lib/ci/default-shell {0}
+
+# Cancel in progress workflows on pull_requests.
+# https://docs.github.com/en/actions/using-jobs/using-concurrency#example-using-a-fallback-value
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  create-conflict-message:
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+
+    steps:
+      - name: Update PR
+        if: github.event.pull_request.mergeable_state == 'conflicting'
+        uses: thollander/actions-comment-pull-request@v2.4.3
+        with:
+          message: |
+            This PR currently has a merge conflict, which is preventing
+            other TACOS jobs from running. Please solve the conflict and
+            then try again.
+          comment_tag: conflict
```

Comment:
`comment_tag` here means that the second time this job runs it will update the same comment with the same content, resulting in no perceivable effect for the user.

... at the same time, this behavior is preferable to what we have at HEAD. Ack, add a TODO comment, and resolve this comment if you don't see a way to do better.

Reply:
https://github.com/thollander/actions-comment-pull-request?tab=readme-ov-file#delete-a-comment

Maybe?

```
   mode =  {{ github.event.pull_request.mergeable_state == 'conflicting' && 'recreate' || 'delete' }}
```

The README suggests that mode:delete will create then delete the comment, but the code suggests otherwise: https://github.com/thollander/actions-comment-pull-request/blob/main/src/main.ts#L160-L162

Reply:
the delete mode is handled in a separate file: https://github.com/thollander/actions-comment-pull-request/blob/main/src/cleanup.ts

Reply:
Added more info to the message, including date and sha of update.